package.json

```diff
@@ -42,22 +43,17 @@
     "@typescript-eslint/parser": "^1.10.2",
     "babel-core": "^7.0.0-bridge.0",
     "babel-eslint": "^10.0.1",
-    "babel-loader": "^8.0.6",
     "eslint": "^5.16.0",
     "eslint-config-airbnb": "^17.1.0",
     "eslint-plugin-import": "^2.17.3",
     "eslint-plugin-jsx-a11y": "^6.2.1",
     "eslint-plugin-react": "^7.13.0",
     "eslint-plugin-react-hooks": "^1.6.0",
-    "html-webpack-plugin": "^3.2.0",
     "jest": "^24.8.0",
+    "microbundle": "^0.11.0",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
-    "ts-loader": "^6.0.2",
-    "typescript": "^3.5.1",
-    "webpack": "^4.33.0",
-    "webpack-cli": "^3.3.4",
-    "webpack-dev-server": "^3.7.1"
```

Comment:
We are using `webpack-dev-server` for `npm run examples:minimal` and `npm run examples:typescript`.
Is there any alternative that would work well along with microbundle?

Reply:
If there were an alternative without webpack-dev-server, we could eliminate babel packages too. That would be awesome.

Reply:
Maybe: https://github.com/thgh/rollup-plugin-serve
Or, [react-scripts](https://github.com/facebook/create-react-app/tree/master/packages/react-scripts)?
(We do have babel dependencies anyway. It's just whether explicit or implicit.)

Reply:
We need the babel packages for jest as far as I know. Did not know about the examples will add it back.

Reply:
Sorry that I didn't pay much attention for windows devices.
Does cross-env help?